glsim/engine.py

```
@@ -263,16 +290,22 @@ def call_method(
             msg = self._post_queue.pop(0)
             self._post_queue.clear()
             self._draining = True
+            print(f"[PostMessage DRAIN] executing {msg['method']} on {msg['address']}")
             try:
                 self.call_method(
                     msg['address'], msg['method'],
                     msg.get('args', []), msg.get('kwargs', {}),
                     sender=msg.get('sender'),
                 )
+                print(f"[PostMessage DRAIN] {msg['method']} completed OK")
             except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)